src/pages/Workshops/index.tsx

```diff
@@ -9,7 +9,7 @@ const Workshops: React.FC = () => {
     <div className="page workshops">
       <Navbar />
       <div className="container workshop-content">
-        <h2>hack.py</h2>
+        <h2 id='top'>hack.py</h2>
```

Comment:
if it works yeah go for it — otherwise the official location for `top` i'd say would be in `src/index.tsx` (change `<div className="app">` to be `<div className="app" id="top">`

Reply:
`src/index.tsx` is definitely a better place to have it, I'll move it there